superduperdb/vector_search/lance.py

```diff
@@ -27,7 +27,10 @@ def __init__(
         index: t.Optional[t.List[str]] = None,
         measure: t.Optional[str] = None,
     ):
-        self.dataset_path = f'.superduperdb/vector_indices/{identifier}.lance'
+        lance_home = os.environ.get(
+            'SUPERDUPERDB_LANCE_HOME', '.superduperdb/vector_indices'
+        )
```

Comment:
We should add a project level configuration?

Reply:
Yes, I have added it, please see if the configuration level is reasonable
https://github.com/SuperDuperDB/superduperdb/pull/1253#discussion_r1386170470

@blythed